test/utils/noderesourcetopologies/noderesourcetopologies.go

```diff
@@ -221,7 +221,14 @@ func checkEqualResourcesInfo(nodeName, zoneName string, resourcesInitial, resour
 }
 
 func checkConsumedResourcesAtLeast(resourcesInitial, resourcesUpdated []nrtv1alpha1.ResourceInfo, required corev1.ResourceList) (bool, error) {
+	_, cpuRequired := required[corev1.ResourceCPU]
```

Comment:
this works and I'm OK with it. Please consider just passing the pod QoS as extra parameter. This will make the code both clearer and more robust, but can require changing quite a few places.

Reply:
Thanks for your point! 
updated the code accordingly.